articles/connections/passwordless/configure-passwordless-connection.md

```diff
@@ -0,0 +1,137 @@
+
+If you choose to extend the OPT expiry time, we recommend increasing the OTP length for security reasons. Otherwise, the malicious party has a longer time frame to try to guess a short code.
+
+6. Disable Sign Ups
+
+If you want to restrict sign-ups and only allow the addition of new users via the API or the Dashboard, click the **Disable Sign Ups** toggle to enable this feature.
+
+When you are done making configuration changes, click **Save**.
+
+## Configure Passwordless using email
+
+1. Navigate to the [Connections > Passwordless](${manage_url}/#/connections/passwordless) page in the [Auth0 Dashboard](${manage_url}/), and enable the Email toggle.
+
+![Enable Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-list.png)
+
+2. Enter the email address you want users to see as the sender in the **From** field, and set the **Subject** line for your email.
+
+![Configure Email Passwordless](/media/articles/connections/passwordless/connections-passwordless-email.png)
+
+::: note
+If you are using a custom template, you must change the **From** value to an address that does *not* use the domain **auth0.com**. Otherwise, the default email template will be sent.
+:::
+
+3. Edit the **Body** of your email. If you have an email template you want to use, you can provide the HTML for it here. Alternatively, you can use Auth0's default template.
+
+The template editor accepts Liquid syntax embedded within the HTML. This allows you to construct parts of your message programmatically. For example, you can reference the `request_language` parameter to change the language of the message:
+
+```text
+{% if request_language contains 'dutch' %}
+ Hier is uw verificatie code: @@password@@
+{% endif %}
+
+{% if request_language contains 'fr-FR' %}
+ Ceci est votre code: @@password@@
+{% endif %}
+```
+
+The following parameters are available to you when defining the email template:
+
+| Parameter | Description |
+| - | - |
+| `password` or `code` | The one-time password to send to the user |
+| `phone_number ` | The user's phone number |
+| `application.name` | The name of the application to which the user is logging in |
+| `request_language` | The requested language to be used for the SMS message's content |
+
+You can also use multiple languages in your emails. To set the language used, call the [`passwordless/start`](/docs/api/authentication/reference#get-code-or-link) endpoint and set the `x-request-language` header to the language of your choice. Otherwise, the language used will be extracted from the `accept-language` header (which is automatically set by your browser).
+
+If at any time, you want to revert your changes, you can do so. You can either **reset to last saved** template, or you can **reset to default**.
+
+4. Enter any **Authentication Parameters** you would like to include in the generated sign-in link. These will be included as query string parameters.
```

Comment:
Give me some brief context and/or point me to a doc to tell me what this means. Pretend I'm an identity newbie. Why would I want to do this?